src/PersonObjects/Person.ts

```diff
@@ -31,35 +35,44 @@ export abstract class Person implements IPerson {
     intelligence: 0,
   };
 
-  mults = defaultMultipliers();
-
-  /** Augmentations */
-  augmentations: PlayerOwnedAugmentation[] = [];
-  queuedAugmentations: PlayerOwnedAugmentation[] = [];
+// Multipliers affecting various actions and attributes, represents skills gained and new speeds / rates because of these skills.
+mults = defaultMultipliers();
 
-  /** City that the person is in */
-  city: CityName = CityName.Sector12;
+/** Augmentations */
+augmentations: PlayerOwnedAugmentation[] = []; // List of augmentations owned by the person
```

Comment:
These two are solid explanatory comments for these fields. 👍They explain the difference between the two in a way that the variable name alone does not convey. Keep these, but I feel like everything else up until here (and after here, until the next comment) should be reverted as already self-explanatory.